docs/windows/BUILD.md

```diff
@@ -0,0 +1,430 @@
+-o dist/windows/pinshare-tray.exe ./cmd/pinshare-tray
+
+# On Windows
+go build -ldflags="-H windowsgui" -o dist\windows\pinshare-tray.exe .\cmd\pinshare-tray
+```
+
+The `-H windowsgui` flag prevents a console window from appearing.
+
+#### 4. IPFS Kubo Binary
+
+```bash
+# Download and extract
+IPFS_VERSION=v0.31.0
+curl -L -o /tmp/kubo.zip \
+  "https://dist.ipfs.tech/kubo/${IPFS_VERSION}/kubo_${IPFS_VERSION}_windows-amd64.zip"
+
+unzip -j /tmp/kubo.zip "kubo/ipfs.exe" -d dist/windows/
+```
+
+On Windows (PowerShell):
+```powershell
+$IPFS_VERSION = "v0.31.0"
+$URL = "https://dist.ipfs.tech/kubo/$IPFS_VERSION/kubo_${IPFS_VERSION}_windows-amd64.zip"
+Invoke-WebRequest -Uri $URL -OutFile kubo.zip
+Expand-Archive -Path kubo.zip -DestinationPath .
+Move-Item kubo\ipfs.exe dist\windows\
+Remove-Item kubo.zip
+Remove-Item -Recurse kubo
+```
+
+## Building the Installer
+
+### Prerequisites
+
+**WiX Toolset must be installed and in PATH.**
+
+Verify:
+```cmd
+candle.exe -?
+light.exe -?
+```
+
+### Build Steps
+
+#### On Windows (Git Bash or CMD)
+
+```bash
+cd installer
+
+# Using shell script (works in Git Bash, delegates to .bat)
+./build-wix6.sh [version]
+
+# Or use batch file directly (CMD/PowerShell)
+.\build-wix6.bat [version]
+```
+
+This uses WiX 4.x/6.x toolset (installed via `dotnet tool install`) to build the MSI installer.
+
+Output: `installer/bin/Release/PinShare-Setup.msi`
+
+#### On macOS/Linux
+
+WiX cannot run natively on macOS/Linux. Options:
+
+1. **CI/CD Pipeline:** Use GitHub Actions with Windows runners (recommended)
+2. **Windows VM:** Copy built binaries to Windows and run `build-wix6.bat`
+3. **Cross-compile binaries locally, build MSI in CI:** The `build-windows.sh` script creates all binaries; the MSI can be built by GitHub Actions
+
+#### Using CI/CD
+
+For automated builds, use GitHub Actions with Windows runners. See `.github/workflows/build.yml` for an example.
+
+## Troubleshooting Build Issues
+
+### CGO Errors
+
+**Error:** `gcc: command not found`
+
+**Linux/macOS Solution:**
+```bash
+# Install MinGW
+sudo apt-get install gcc-mingw-w64-x86-64  # Debian/Ubuntu
+brew install mingw-w64                      # macOS
+```
+
+**Windows Solution:**
+```
+Install TDM-GCC or MinGW-w64, add to PATH
+```
+
+**Error:** `undefined reference to...` (SQLite linking)
+
+**Solution 1:** Ensure CGO is enabled
+```bash
+export CGO_ENABLED=1
+export CC=x86_64-w64-mingw32-gcc  # Linux
+```
+
+**Solution 2:** Use pure-Go SQLite
+```bash
+# In go.mod, replace:
+# github.com/mattn/go-sqlite3
+# with:
+# modernc.org/sqlite
+
+# Then build without CGO
+CGO_ENABLED=0 go build ...
+```
+
+### IPFS Download Errors
+
+**Error:** `curl: (6) Could not resolve host`
+
+**Solution:** Check internet connection, try:
+```bash
+# Use wget instead
+wget https://dist.ipfs.tech/kubo/v0.31.0/kubo_v0.31.0_windows-amd64.zip
+```
+
+**Error:** `unzip: command not found`
+
+**Linux Solution:**
+```bash
+sudo apt-get install unzip
+```
```

Comment:
It's not clear under what circumstances this is likely or anticipated. I think we should probably just remove it.

```suggestion
```